zebra-state/src/service/non_finalized_state.rs

```diff
@@ -289,16 +289,13 @@ impl NonFinalizedState {
         None
     }
 
-    /// Returns the `block` at a given height or hash in the best chain.
-    pub fn best_block(&self, hash_or_height: HashOrHeight) -> Option<Arc<Block>> {
+    /// Returns the [`Block`] at a given height or hash in the best chain.
```

Comment:
Minor: Should this be `ContextuallyValidBlock`?

```suggestion
    /// Returns the [`ContextuallyValidBlock`] at a given height or hash in the best chain.
```